packages/grafana-ui/src/components/FormGroup/FormGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { SFC } from 'react';
+import { Label } from '..';
+
+export interface Props {
+  label: string;
+  inputProps: {};
```

Comment:
What do you think about `Props` interface extending `React.InputHTMLAttributes` instead of using `inputProps`?